src/Service.Tests/Unittests/SerializationDeserializationTests.cs

```diff
@@ -305,6 +369,45 @@ private void InitializeObjects()
             _databaseStoredProcedure.StoredProcedureDefinition.Parameters.Add("Id", _parameterDefinition);
         }
 
+        /// <summary>
+        /// During serialization we add TypeName to the DatabaseObject based on its type.
+        /// If the code refactor then this change will catch that as GraphQL workload would be impacted.
+        /// We will have artifacts with TypeName with old project directory/ location of this class, hence during deserialization if object is not in
+        /// that location it will fail.
+        /// [Note] : is this test fails notify the graphql workload team.
```

Comment:
```suggestion
        /// [Note] : If this test fails, notify the GraphQL workload team.
```

Since this project is open source, this statement doesn't translate well or at the very least is unactionable by an external contributor. I'd suggest that devs should open a GitHub issue and add a specific tag that you specify -> when they encounter a failure in this test.

Reply:
e.g. tag such as `GraphQLWorkloadBreakingChange` or `SDK-Compat`

Reply:
updated the note section as suggested.